recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -8,27 +8,108 @@ inherit cmake pythonnative
+BEEROCKS_BH_WIRE_IFACE_raspberrypi-rdk-broadband = "erouter0"
+#
+# RDKB specific
+# "ZeroMG" has incompatible license for RDKB so use NNG
+MSGLIB_rdk ?= "mq-nng"
+
+BEEROCKS_BRIDGE_IFACE_rdk ?= "brlan0"
+#
+# FIXME: There is a bug in script printing status
+#        if log files paths are not standard
+# So temporary disable redefinition to use defaults
+# BEEROCKS_LOG_FILES_ENABLED_rdk ?= "false"
+# BEEROCKS_LOG_SYSLOG_ENABLED_rdk ?= "true"
+# BEEROCKS_LOG_FILES_AUTO_ROLL_rdk ?= "false"
+# BEEROCKS_LOG_FILES_PATH_rdk ?= "/rdklogs/logs"
+# BEEROCKS_LOG_FILES_SUFFIX_rdk ?= ".txt.0"
+#
+
+PACKAGECONFIG[debug] = "-DCMAKE_BUILD_TYPE=Debug,-DCMAKE_BUILD_TYPE=Release,,"
+
+PACKAGECONFIG[mq-zmq] = "-DMSGLIB=zmq,,zeromq,"
+PACKAGECONFIG[mq-nng] = "-DMSGLIB=nng,,nng,"
+PACKAGECONFIG[mq-none] = "-DMSGLIB=None,,nng,"
+
+PACKAGECONFIG[bwl-nl80211] = "-DBWL_TYPE=NL80211,,wpa-supplicant libnl,wpa-supplicant"
+PACKAGECONFIG[bwl-dummy] = "-DBWL_TYPE=DUMMY,,wpa-supplicant libnl,wpa-supplicant"
+PACKAGECONFIG[bwl-dwpal] = "-DBWL_TYPE=DWPAL,,libnl ubus uci wav-dpal,"
 
 EXTRA_OECMAKE = "-DTARGET_PLATFORM=${TARGET_PLATFORM} \
-                 -DBWL_TYPE=${BWL_TYPE} \
                  -DUSE_USER_TMP_PATH=0 \
                  -DCMAKE_INSTALL_PREFIX:PATH=${INSTALL_PREFIX} \
-                 -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
-                 -DMSGLIB=zmq"
+                 -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON \
+                 "
+
+def oecmake_var (name, d):
+    v = d.getVar(name, True)
+    return '-D{}={}'.format(name, v) if v else ''
+
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_BRIDGE_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_BH_WIRE_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_WLAN1_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_WLAN2_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_WLAN3_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_LOG_FILES_ENABLED', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_LOG_FILES_PATH', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_LOG_FILES_SUFFIX', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_LOG_FILES_AUTO_ROLL', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_LOG_STDOUT_ENABLED', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_LOG_SYSLOG_ENABLED', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_HOSTAP_WLAN1_CTRL_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_HOSTAP_WLAN2_CTRL_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_HOSTAP_WLAN3_CTRL_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_WPA_SUPPLICANT_WLAN1_CTRL_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_WPA_SUPPLICANT_WLAN2_CTRL_IFACE', d)}"
+EXTRA_OECMAKE += "${@oecmake_var('BEEROCKS_WPA_SUPPLICANT_WLAN3_CTRL_IFACE', d)}"
+
+do_configure_prepend() {
+    install -d ${D}${datadir}/cmake/Modules
+    install -m 0644 ${S}/cmake/Findnl-3.cmake ${D}${datadir}/cmake/Modules/
+    install -m 0644 ${S}/cmake/Findnl-genl-3.cmake ${D}${datadir}/cmake/Modules/
+}
```

Comment:
Why is this needed? prplmesh CMakeLists.txt includes ${S}/cmake into its search path, so it shouldn't be needed to do this.

Reply:
Will be fixed in https://jira.prplfoundation.org/browse/PPM-282